frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,213 @@
-    @media screen and (max-width: 800px) {
-      min-width: ${mobileDrawerWidth};
-    }
-  }
-  `}
-`;
-
-const DrawerHeader = styled(Grid)`
-  ${({ theme }) => `
-    justify: flex-start;
-    direction: row;
-    ${theme.mixins.toolbar};
-  `}
-`;
-
-const ExpandLessIcon = styled(ExpandLess)`
-  ${({ theme }) => `
-  color: ${theme.palette.primary.main};
-  `}
-`;
-
-const ExpandMoreIcon = styled(ExpandMore)`
-  ${({ theme }) => `
-  color: ${theme.palette.primary.main};
-  `}
-`;
-
-const GroupHeading = styled(Typography)`
-  ${({ theme }) => `
-  color: ${theme.palette.primary.main};
-  padding-top: 0.25rem;
-  font-weight: bolder;
-  `}
-`;
-
-const TrendingIcon = styled(TrendingUpIcon)`
-  fontsize: 20px;
-  marginleft: 10px;
-`;
-
-const NavigationLink = styled(RouterLink)`
-  ${({ theme }) => `
-  color: ${theme.palette.text.secondary};
-  `}
-`;
-
-interface HeaderProps {
-  onNavigate: () => void;
-}
-
-const Header: React.FC<HeaderProps> = ({ onNavigate }) => {
-  return (
-    <DrawerHeader container flex-wrap="wrap">
-      <Grid item>
-        <NavigationLink to="/" onClick={onNavigate} data-qa="logo">
-          <Logo />
-        </NavigationLink>
-      </Grid>
-      <Grid item>
-        <Grid container justify="center" direction="column">
-          <Typography
-            style={{ lineHeight: "1.3", margin: "-5px 0px 10px 10px" }}
-            component="span"
-            color="primary"
-            data-qa="title"
-          >
-            <NavigationLink to="/" onClick={onNavigate}>
-              <GroupHeading>clutch</GroupHeading>
-            </NavigationLink>
-            <Typography variant="caption">Welcome {userId()}!</Typography>
-          </Typography>
-        </Grid>
-      </Grid>
-    </DrawerHeader>
-  );
-};
-
-const Divider = styled(MuiDivider)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.accent.main};
-  `}
-`;
+// sidebar
+const drawerWidth = "100px";
+
+const DrawerPanel = styled(MuiDrawer)({
+  flexShrink: 0,
+  width: `${drawerWidth}`,
+  "div[class*='MuiDrawer-paper']": {
+    width: `${drawerWidth}`,
+    top: "64px",
+    backgroundColor: "linear-gradient(0deg, #FFFFFF, #FFFFFF), #FFFFFF",
+    boxShadow: "0px 5px 15px rgba(53, 72, 212, 0.2)",
+  },
+});
+
+// sidebar groupings
+const GroupList = styled(List)({
+  padding: "0px",
+});
+
+const GroupListItem = styled(ListItem)({
+  flexDirection: "column",
+  height: "82px",
+  padding: "16px 8px 0px 8px",
+  "&:hover": {
+    backgroundColor: "#E7E7EA",
+  },
+  "&:active": {
+    backgroundColor:
+      "linear-gradient(0deg, rgba(255, 255, 255, 0.85), rgba(255, 255, 255, 0.85)), #0D1030",
+  },
+  "&.Mui-selected": {
+    backgroundColor: "#EBEDFB",
+    ".MuiAvatar-root": {
+      color: "#FFFFFF",
+      background: "#3548D4",
+    },
+  },
+});
+
+const Avatar = styled(MuiAvatar)({
+  color: "rgba(13, 16, 48, 0.38)",
+  background: "rgba(13, 16, 48, 0.12)",
+  height: "36px",
+  width: "36px",
```

Comment:
avatar is too large and is also causing inconsistent top and bottom padding (content not vertically centered)

![image](https://user-images.githubusercontent.com/4712430/101806419-8cd4c280-3ad9-11eb-9a56-d0b5d7f09457.png)

Reply:
thanks for catching! width & height should be 24px. also fixed some of the padding

<img width="111" alt="Screen Shot 2020-12-11 at 10 32 31 AM" src="https://user-images.githubusercontent.com/39421794/101922605-4d22df00-3b9c-11eb-9023-915bce5d5858.png">

<img width="111" alt="Screen Shot 2020-12-11 at 10 33 06 AM" src="https://user-images.githubusercontent.com/39421794/101922624-53b15680-3b9c-11eb-9657-5774208cc3f8.png">